src/discussions/comments/CommentsView.jsx

```diff
@@ -41,12 +45,19 @@ function CommentsView({ intl }) {
   return (
     <div className="discussion-comments d-flex flex-column w-100 ml-3">
       <div className="mb-2">
+        { thread.closed && BANNER_TYPE.closed(intl.formatMessage(bannerMessages.closed)) }
+        { thread.abuseFlagged && BANNER_TYPE.reported(intl.formatMessage(bannerMessages.contentReported)) }
+        { thread.pinned && BANNER_TYPE.pinned(intl.formatMessage(bannerMessages.pinned)) }
```

Comment:
These banners are in the wrong place. 
The only banner I see in the mockups is for closed threads. If a thread is pinned or reported, it gets a banner in the preview on the left side.